qsimcirq/qsim_circuit.py

```diff
@@ -127,11 +127,8 @@ def __eq__(self, other):
 
   def _resolve_parameters_(self, param_resolver: cirq.study.ParamResolver):
 
-    qsim_circuit = super()._resolve_parameters_(param_resolver)
-
-    qsim_circuit.device = self.device
-
-    return qsim_circuit
+    return QSimCircuit(
```

Comment:
Looks good to me. Perhaps empty line 129 should be deleted.

Reply:
Done.